updates/stable.json

```diff
@@ -45,19 +45,29 @@
       }
     },
     {
-      "version": "34.20210821.3.0",
+      "version": "34.20210904.3.0",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "34.20210904.3.0",
+      "version": "34.20210919.3.0",
+      "metadata": {
+        "rollout": {
+          "duration_minutes": 2880,
+          "start_epoch": 1633618800,
+          "start_percentage": 0.0
+        }
+      }
+    },
+    {
+      "version": "34.20210919.3.0",
       "metadata": {
         "rollout": {
           "duration_minutes": 2880,
-          "start_epoch": 1632236400,
+          "start_epoch": 1633568400,
           "start_percentage": 0.0
         }
       }
```

Comment:
Looks like the rollout is duplicated here. There should only be one.

Reply:
Thanks @jlebon fixed!

Reply:
Added a couple validations in #385, including for duplicate rollouts.